zipkin-storage/cassandra3/src/main/java/zipkin/storage/cassandra3/CassandraUtil.java

```diff
@@ -101,7 +101,7 @@ public static int durationIndexBucket(long ts_micro) {
       annotationKeys.add(request.serviceName + ":" + a);
     }
     for (Map.Entry<String, String> b : request.binaryAnnotations.entrySet()) {
-      annotationKeys.add(request.serviceName + ":" + b.getKey() + ":" + b.getValue());
+      annotationKeys.add(request.serviceName + ";" + b.getKey() + ";" + b.getValue());
```

Comment:
cc @michaelsembwever on "cassandra3" I had to change delimiters of binary annotations to ';' to avoid a LIKE clash with annotations (which are delimited with ':')